### PR TITLE
Profile: Show Tips Sent as well as Tips Recieved

### PR DESCRIPTION
**Pull request by nemaniarjun**
- Removed Tips Function
- Added `Tips Sent` HTML to profile template
- Made all neccesary related changes

Fixes: https://github.com/gitcoinco/web/issues/1727

##### Description
Profile: Show Tips Sent

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
ui

### CODE REVIEW

File: app/dashboard/models.py [hunk 1]
```diff
@@ -1521,16 +1521,10 @@ def bounties(self):
             bounties = bounties | Bounty.objects.filter(interested=interested, current_bounty=True)
         bounties = bounties | Bounty.objects.filter(pk__in=fulfilled_bounty_ids, current_bounty=True)
         bounties = bounties | Bounty.objects.filter(bounty_owner_github_username__iexact=self.handle, current_bounty=True) | Bounty.objects.filter(bounty_owner_github_username__iexact="@" + self.handle, current_bounty=True)
-        bounties = bounties | Bounty.objects.filter(github_url__in=[url for url in self.tips.values_list('github_url', flat=True)], current_bounty=True)
+        bounties = bounties | Bounty.objects.filter(github_url__in=[url for url in self.received_tips.values_list('github_url', flat=True)], current_bounty=True)
```
[the remark this answers is not in the document]
Comment (nemaniarjun) [reply app/dashboard/models.py]: Well it is shorter than the line above it...

**Comment by nemaniarjun** [comment]
Oops! Added temp `a.py` file accidentally resulting in the extra comment by stickler-ci

**Comment by thelostone-mc** [comment]
mind adding a screenshot and fixing eslint errors?

**Comment by nemaniarjun** [comment]
@thelostone-mc Done!
The screenshot (same as the one I posted on the issue):

**Comment by vs77bb** [comment]
@thelostone-mc @mbeacom This good for merge / payout?

**Comment by thelostone-mc** [comment]
@mbeacom had a few concerns on this approach ! Could we wait it out ? 😅

**Comment by thelostone-mc** [comment]
Closing this out as this has been integrated into the profile ! Thanks @nemaniarjun though ^_^

**Pull request closed without merging**